pages/examples/index.html

```diff
@@ -66,6 +66,25 @@
                         <!-- <img class="footerImage" src="images/logoGray.svg" title="Standalone Visualization"> -->
                     </div>
                 </div>
+                <div class='sampleContainer standalone'>
+                    <div class='sampleHeader'>
```

Comment:
Hope these PNG images do not have a copyrights issue :)

Reply:
Taken from https://www.svgrepo.com/. Free to download open licence svgs.

![image](https://github.com/user-attachments/assets/44b8b3af-e88c-4efd-b82e-c396b14d854b)